pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -258,3 +258,7 @@ func (cc *cadvisorClient) HasDedicatedImageFs() (bool, error) {
 	}
 	return imageFsInfo.Device != rootFsInfo.Device, nil
 }
+
+func (cc *cadvisorClient) GetFsInfoByFsUUID(uuid string) (cadvisorapiv2.FsInfo, error) {
+	return cc.GetFsInfoByFsUUID(uuid)
```

Comment:
Why the function is calling itself? Did I miss anything?

Reply:
Removed this function to expose the embedded one in `manager.Manager`.